experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -472,7 +472,7 @@ describe("PageSummary", () => {
     render(<Subject mocks={[mock, mutationMock]} />);
     await launchFromDraftToReview();
     await waitFor(() =>
-      expect(screen.getByTestId("submit-error")).toBeInTheDocument(),
+      expect(screen.getByTestId("warning-submit-error")).toBeInTheDocument(),
```

Comment:
nit: replace the `expect(...).toBeInTheDocument()` with `screen.getByTestId(...)`

`getByTestId` will throw if it does not find the selector.